src/main/resources/db/changelog/db.changelog-master.yaml

```diff
@@ -45,7 +45,12 @@ databaseChangeLog:
                     constraints:
                       primaryKey: True
                       primaryKeyName: client_pkey
-                    name: client_id;
```

Comment:
can you please create a new changeset for this task ? it's a good practice to never update existing changesets after it's merged